include/deal.II/base/vectorization.h

```diff
@@ -526,6 +527,13 @@ class VectorizedArray
     data = base_ptr[offsets[0]];
   }
 
+  DEAL_II_ALWAYS_INLINE
+  void
+  gather(const std::array<Number *, width> &base_ptr, const unsigned int i)
```

Comment:
base_ptr is not a pointer, though. Rename? Why ``i`` and not ``offset``?

Reply:
I changed the names.